libs/pika/mpi_base/include/pika/mpi_base/mpi_environment.hpp

```diff
@@ -81,12 +60,12 @@ namespace pika { namespace util {
 
 # include <pika/config/warnings_prefix.hpp>
 
-namespace pika { namespace util {
-        struct PIKA_EXPORT mpi_environment
-        {
-            static bool check_mpi_environment(runtime_configuration const& cfg);
-        };
-}}    // namespace pika::util
+namespace pika::mpi {
+    struct PIKA_EXPORT environment
+    {
+        static bool check_environment(runtime_configuration const& cfg);
+    };
+}    // namespace pika::mpi
```

Comment:
Do you know if we need this dummy definition? We may be ok with removing it completely when MPI is disabled, assuming calls to it are ifdefed out. Especially given the comment that `check_environment` is unused, we may be able to remove it.